webapp/sources/rudder/rudder-core/src/main/scala/com/normation/cfclerk/xmlparsers/SectionSpecParser.scala

```diff
@@ -69,16 +68,6 @@ class SectionSpecParser(variableParser:VariableSpecParser) extends Loggable {
            */
           val variableNames = root.getAllVariables.map( _.name )
 
-          /*
-           * check that all variable and section names are unique
-           */
-          val allVarUniqueName = checkUniqueness(variableNames) {
-            "At least two variables have the same name (case unsensitive), what is forbiden: "
-          }
-          val allSectionUniqueName = checkUniqueness(root.getAllSections.map(_.name)) {
-            "At least two sections have the same name (case unsensitive), what is forbiden: "
-          }
```

Comment:
these restriction can be lift because we now have an unique identifier for each component ?